### DISCUSSION TITLE
按照说明中设置 Cookie 失败怎么办？

### DISCUSSION THREAD

**Discussion by breathiness** [yutto-dev/yutto#64] [Q&A] [answered]
这个命令我是输错了吗？

![图片](https://user-images.githubusercontent.com/11361630/154849128-a81249b3-c3ca-40b9-a936-16f0d6693e0b.png)

**Comment by SigureMo** [comment, marked as answer]
emmmm，`<url>` 要改成你要下载的链接啊……

**Reply by breathiness** [reply]
这样为了下载最高画质的视频，我每次下载都需要输入 Cookie ？感觉不太合理啊。而且 URL 在中间，也不好用 alias 的方式自动化。有什么方法可以不让我每次都输入吗？

**Reply by SigureMo** [reply]
url 不必在中间……

而且我也给出了设置 alias 的例子……

https://github.com/SigureMo/yutto#%E8%87%AA%E5%AE%9A%E4%B9%89%E5%91%BD%E4%BB%A4%E5%88%AB%E5%90%8D

**Reply by breathiness** [reply]
原来如此，看漏了。我的我的😂
